src/wafp/fuzzers/catalog/schemathesis/__init__.py

```diff
@@ -24,6 +33,10 @@ def get_entrypoint_args(
                 args.extend(["-H", f"{key}: {value}"])
         if ssl_insecure:
             args.extend(["--request-tls-verify=false"])
+        if self.send_report_requested and self.saas_token:
+            self.logger.info("Report gonna be sent to schemathesis.io")
+            args.extend(["--report"])
+            args.extend([f"--schemathesis-io-token={self.saas_token}"])
```

Comment:
Will it be available in logs then? Maybe just via env vars?

Reply:
Shouldn't. I didnt find any place where this run commands are logged.

Reply:
Cool!